trace-utils/src/span/v05/mod.rs

```diff
@@ -82,8 +209,47 @@ mod tests {
             )]),
             metrics: HashMap::from([(BytesString::from("metrics_field"), 1.1)]),
             meta_struct: HashMap::new(),
-            span_links: vec![],
-            span_events: vec![],
+            span_links: vec![SpanLinkBytes {
```

Comment:
/suggestion

- would be nice to have test case for multiple span links
- a complex attributes (remember we do some flattening for array and maps), I'm not sure at this point we have them already or we have to do ourselves
   eg 
```
{"key": [[1,2], ["3", "4"]]} to {"key.0.0": "1", "key.0.1": 2, "key.1.0": 3, "key.1.1":4}
```